### ISSUE TITLE
Pokemon ROMS

### ISSUE THREAD

**Issue by ImSoIronik** [Cattn/gba#133]
When going to singleplayer on GBA 2.0, the pokemon rom hacks do not work (radical red, altered emerald, etc.). I don't know if this is only me but they do not work. 

P.S I am on chromebook.

**Comment by Cattn** [comment]
Yeah, I really needa go through and remove all those old broken gba games

**Comment by Cattn** [comment]
let me know if they worked on gba, but not GBA2.0

**Comment by ImSoIronik** [comment]
they worked on GBA but not GBA 2.0. I had to find a website and download
the exact game that was used on GBA just to play my save file of radical
red and it wasn't easy

**Comment by Cattn** [comment]
Can you send me the share link in GBA2.0 to the game in specific?

**Comment by ImSoIronik** [comment]
https://math-study.github.io/gba/main/play/index.html?console=gba&game=Pokemon-Radical-Red-v3

**Comment by Cattn** [comment]
Thanks, I see the issue

**Comment by ImSoIronik** [comment]
Awesome, thanks.
